iquip/apps/scheduler.py

```diff
@@ -59,6 +80,21 @@ def __init__(self, parent: Optional[QWidget] = None):
         layout.addWidget(self.queueView)
 
 
+class ExperimentListView(QListView):
+    """Customized QListView class to detect right-click input.
```

Comment:
I think this description is too minor.

How about writing that this list view contains the running experiments.

Reply:
Please check it! @giwon2004

Reply:
I did this under this line! Github only shows the modification on that exact line :(

Reply:
Oh, you added a description under this line!
I'm sorry I didn't check well.